js/digifinex.js

```diff
@@ -41,11 +42,18 @@ module.exports = class digifinex extends Exchange {
                 'fetchCurrencies': true,
                 'fetchDepositAddress': true,
                 'fetchDeposits': true,
+                'fetchFundingHistory': false,
```

Comment:
I think this endpoint might provide it https://docs.bitfinex.com/reference/ws-auth-funding-trades

Reply:
That's the wrong exchange